test/remote/gateways/remote_global_collect_test.rb

```diff
@@ -55,18 +55,17 @@ def test_successful_purchase_with_more_options
     assert_equal 'Succeeded', response.message
   end
 
-  # When approval is required, purchase will not make a capture request
+  # When approval is required, `purchase` will also make a `capture` request
   def test_successful_purchase_with_requires_approval_true
     options = @options.merge(requires_approval: true)
 
     response = @gateway.purchase(@amount, @credit_card, options)
     assert_success response
     assert_equal 'Succeeded', response.message
-    assert_equal 'PENDING_APPROVAL', response.params['payment']['status']
-    assert_equal 'PENDING_MERCHANT', response.params['payment']['statusOutput']['statusCategory']
+    assert_equal 'CAPTURE_REQUESTED', response.params['payment']['status']
   end
 
-  # When approval is not required, purchase will make a capture request
+  # When approval is required, `purchase` will request capture via `auth`
```

Comment:
Shouldn't this still say "not required"?

Perhaps add some more clarification here saying that the `purchase` request will send an `auth` request and then perform an automatic capture that does not require an explicit `capture` request?

Reply:
Whoops, good catch on that comment. I've clarified that comment, feel free to have another looksie.

Reply:
@fatcatt316 Hmm, I'm not seeing any changes yet. Maybe try pushing again?

Reply:
@leila-alderman 🤦‍♂ Pushed now.